stellargraph/mapper/full_batch_generators.py

```diff
@@ -157,8 +157,8 @@ def flow(self, node_ids, targets=None):
         Args:
             node_ids: an iterable of node ids for the nodes of interest
                 (e.g., training, validation, or test set nodes)
-            targets: a 1D or 2D array of numeric node targets with shape `(len(node_ids)`
-                or (len(node_ids), target_size)`
+            targets: a 1D or 2D array of numeric node targets with shape `(len(node_ids),)`
```

Comment:
Thanks for noticing the bugs here.

Unfortunately this doc string was wrong originally in another way too: the doc strings are reStructuredText formatting, not Markdown, and so code/literal formatting has to be a double backtick (https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#inline-markup):

```suggestion
            targets: a 1D or 2D array of numeric node targets with shape ``(len(node_ids),)``
```

and similarly on the next line and for both of the instances in `FullBatchLinkGenerator.flow`.

This can be observed by the *italics* rather than `fixed width` in https://stellargraph--1438.org.readthedocs.build/en/1438/api.html#stellargraph.mapper.FullBatchLinkGenerator.flow

Reply:
Thanks -- sorry for missing the changes in `FullBatchNodeGenerator`. I've gone ahead and applied the double backticks to the rest of the classes that I could find (by searching for ``` `(len ```) in the code base.

Reply:
Good idea. It looks like there's been quite a bit of copy paste 👍 

> sorry for missing the changes in FullBatchNodeGenerator

Not a problem at all 😄